Tests/SwiftParserTest/Declarations.swift

```diff
@@ -702,7 +702,7 @@ final class DeclarationTests: XCTestCase {
       }
       """,
       diagnostics: [
-        DiagnosticSpec(locationMarker: "EXPECTED_DECL", message: "Expected declaration after '{' in struct"),
+        DiagnosticSpec(locationMarker: "EXPECTED_DECL", message: "Expected declaration in struct"),
```

Comment:
I *think* I agree with doing this, especially since the location would be the "after" anyway (though I suppose not all IDEs display the actual column).